File: sevenvideoview/src/main/java/com/example/sevenvideoview/LogUtil.java
package com.example.sevenvideoview;

import android.util.Log;

/**
 * log工具.
 */
public class LogUtil {

    private static final String TAG = "SevenVideoPlayer";

    public static void d(String message) {
        Log.d(TAG, message);
    }

    public static void i(String message) {
        Log.i(TAG, message);
    }

    public static void e(String message, Throwable throwable) {
        Log.e(TAG, message, throwable);
    }
}

File: sevenvideoview/src/main/java/com/example/sevenvideoview/LogUtil.kt
package com.example.sevenvideoview

import android.util.Log

/**
 * log工具.
 */
object LogUtil {
    private const val TAG = "SevenVideoPlayer"

    fun d(message: String) {
        Log.d(TAG, message)
    }

    fun i(message: String) {
        Log.i(TAG, message)
    }

    fun e(message: String?, throwable: Throwable?) {
        Log.e(TAG, message, throwable)
    }
}
